test/3x/torch/quantization/weight_only/test_autoround.py

```diff
@@ -16,18 +15,28 @@
 )
 from neural_compressor.torch.utils import logger
 
+torch.backends.__allow_nonbracketed_mutation_flag = True
+from neural_compressor.torch.algorithms.weight_only.autoround import get_dataloader
+
 try:
     import auto_round
+    from auto_round.export.export_to_itrex.model_wrapper import WeightOnlyLinear
 
-    AUTO_ROUND_VERSION_0_11 = Version("0.11")
-
-    auto_round_version = auto_round.__version__.split("+")[0]
-    auto_round_version = Version(auto_round_version)
     auto_round_installed = True
 except ImportError:
     auto_round_installed = False
 
 
+def run_fn(model, dataloader):
+    for data in dataloader:
+        if isinstance(data, tuple) or isinstance(data, list):
```

Comment:
@torch.no_grad() or  with torch.no_grad() is required

Reply:
Thanks, updated https://github.com/intel/neural-compressor/pull/1810/commits/54837622aaf62b7c84eae8ca9a1a935ba164dca2